usr/lib/cca_stdll/cca_specific.c

```diff
@@ -3325,6 +3325,54 @@ CK_RV token_specific_set_attrs_for_new_object(STDLL_TokData_t *tokdata,
 
     return ret;
 }
+
+static CK_BBOOL ccatok_token_is_cpacf_exportable(const CK_BYTE *token)
```

Comment:
Please also pass the token size to this function, and check if the token is large enough for CCA_ECC_TOKEN_KEYUSAGE_OFFSET.

Reply:
Done.

Reply:
Done.